### DISCUSSION TITLE
Data import from zenodo

### DISCUSSION THREAD

**Discussion by Sandipriz** [weecology/DeepForest#450] [General]
Hello everyone,

I am trying to run code from tree crown detection using deep learning as provided in[ The Environmental AI book](https://acocac.github.io/environmental-ai-book/forest/modelling/forest-modelling-treecrown_deepforest.html), but got stuck in the very first step of data fetching from zenodo.

I have provided a screenshot of an issue encountered to find the solution.
![git_issue](https://github.com/weecology/DeepForest/assets/74917376/99f67277-d7f1-481a-8c37-ee0c387d2aad)

Any suggestion will be helpful.

**Comment by ethanwhite** [comment]
@Sandipriz - welcome to the DeepForest queue :wave: 

Unfortunately we don't maintain either the Environmental AI book or the `intake_zenodo_fetcher` or the data they are using it to download, so this issue belongs in one of those repos:

* AI book: https://github.com/alan-turing-institute/environmental-ds-book
* Intake Zenodo fetcher: https://github.com/ESM-VFC/intake_zenodo_fetcher

UPDATE: Actually,  @Sandipriz - it looks like maybe you're looking at an outdated version of the Environmental AI Book. Here's the same page for the newer version, which uses a different approach for loading the data https://edsbook.org/notebooks/gallery/15d986da-2d7c-44fb-af71-700494485def/notebook.html 

UPDATE 2: I can confirm that the data import in the up-to-date version of the Environmental AI Book works.

**Comment by ethanwhite** [comment]
@Sandipriz - see updates in my comment above. Looks like if you switch to the new version of the Environmental AI Book everything should work!

**Reply by Sandipriz** [reply]
Thanks, I will check and let you know.
